projects/packages/search/src/instant-search/components/search-results.jsx

```diff
@@ -213,82 +214,90 @@ class SearchResults extends Component {
 	render() {
 		return (
 			<div
-				aria-hidden={ this.props.isLoading === true }
-				className="jetpack-instant-search__search-results"
+				className={ classNames( 'jetpack-instant-search__search-wrapper', {
```

Comment:
This is a nitty suggestion that can be handled in a follow-up PR.

```suggestion
				className={ classNames( 'jetpack-instant-search__search-results-wrapper', {
```

Reply:
Yes! I really got the uncertainty to use the `search-wrapper` or the `search-results-wrapper` as the class name. Even though I thought the latter one looked like the descendant of `search-results` in the HTML structure, it seems better with your suggestion.

Reply:
I will make this update on the following fixes.